### ISSUE TITLE
Error msg in the console

### ISSUE THREAD

**Issue by fbgallet** [mlava/workspaces#11]
If you enter a few characters in Roam search bar, you will see that Workspaces extensions sends an error in the console:

![image](https://github.com/mlava/workspaces/assets/74436347/9b8fd68b-9f65-45fd-ba71-d40897ee3e6c)

**Comment by mlava** [comment]
Hey @fbgallet 
I can't reproduce this.
Is this still happening for you?

**Comment by fbgallet** [comment]
Yes, I think it happens whenever I type enough characters in the search bar, so the "New page: ..." line disappear and is replaced by the first matching page.

**Comment by mlava** [comment]
I've now reproduced.
Will work on sorting it out.
Thanks for reporting!
